mount.c

```diff
@@ -160,12 +162,16 @@ void mount_entries(const char *root, const struct mount_entry *mounts, size_t nm
 		   done prior to calling mount_entries. This is done to avoid EBUSY when
 		   mounting something with the same source and target as one of our
 		   default mounts. Right now, the only case where this applies is the
-		   automatic /proc remount. */
+		   automatic /proc remount. TODO UPDATE THIS???*/
 		size_t targetlen = strlen(target);
 		if (strcmp(mnt->source, "proc") == 0 && strcmp(target + targetlen - 5, "/proc") == 0) {
 			umount2(target, MNT_DETACH);
 		}
-
+    // TODO This is maybe needed here?
```

Comment:
Yes, I'd assume so. Otherwise, an explicit `bst --mount cgroup,/sys/fs/cgroup,cgroup2` would fail for no reason.